pyan/analyzer.py

```diff
@@ -1656,7 +1717,7 @@ def collapse_inner(self):
         # What about incoming uses edges? E.g. consider a lambda that is saved
         # in an instance variable, then used elsewhere. How do we want the
         # graph to look like in that case?
-        
+
         # BUG: resolve relative imports causes (RuntimeError: dictionary changed size during iteration)
```

Comment:
Do you know if this is still the case? (Just curious, since this PR changes the implementation of that feature.)

Reply:
We actually now can resolve those incoming uses edges

Reply:
Ah, I meant the `BUG: resolve relative imports causes...`. Should probably have included a line number there. :)

The question was, do you know if we still need to `for name in list(self.nodes)` instead of just `for name in self.nodes`?

In any case the code is fine like that, it's just the comment that would be nice to fix if it's no longer up to date. But I think we can merge this PR already - that can be fixed later.